resources/ctia-default.properties

```diff
@@ -4,7 +4,7 @@ ctia.store.sql.db.classname=org.h2.Driver
 ctia.store.sql.db.subprotocol=h2
 ctia.store.sql.db.subname=/tmp/ctia-h2-db;DATABASE_TO_UPPER=false
 ctia.store.sql.db.delimiters=
-ctia.store.es.uri=http://localhost:9200
+ctia.store.es.uri=http://192.168.99.100:9200
 ctia.store.es.indexname=ctia
-ctia.producer.es.uri=http://127.0.0.1:9200
+ctia.producer.es.uri=http://192.168.99.100:9200
```

Comment:
Is the IP address change intentional?  Maybe you should set the IP in your local ctia.properties.  If it is intentional, I recommend that you make this change in a separate PR.